src/main/kotlin/racingcar/model/InputParser.kt

```diff
@@ -0,0 +1,24 @@
+package racingcar.model
+
+const val CAR_NAME_REGEX = "^[A-Za-z0-9]{1,5}(,[A-Za-z0-9]{1,5})*$"
+
+class InputParser {
+
+    fun parseCar(input: String): List<String> {
+        if (!validateCar(input)) {
+            throw IllegalArgumentException("Invalid car input")
+        }
+        return input.split(",")
+    }
+
+    fun parseTime(input: String): Int {
+        val result = input.toIntOrNull()
+        return result ?: throw IllegalArgumentException("Invalid time input")
+    }
+
+    private fun validateCar(input: String): Boolean {
+        return Regex(CAR_NAME_REGEX).matches(input)
+    }
```

Comment:
`validateCar()`에서 Boolean 값을 반환하도록 구현하셨는데,
Boolean을 반환하지 않고 함수 내부에서 throw로 예외를 던지면 어떨까요?
그렇게 하면 `parseCar()`에서 if문으로 조건을 확인하지 않고
함수를 호출만 해도 되어 코드가 더 간결해질 것 같습니다!

```suggestion
    private fun validateCar(input: String){
        if (!Regex(CAR_NAME_REGEX).matches(input)) throw IllegalArgumentException("Invalid car input")
    }
```

Reply:
그러네요.. 저번에도 똑같이 잘못해서 고쳤었는데, 감사합니다!